webapp/src/pages/Settings.tsx

```diff
@@ -320,6 +347,71 @@ const Settings: React.FC = () => {
     />
   );
 
+  const displayStringOnlyFields = (
+    field: string,
+    value: string | null,
+    config?: SubConfigKeys
+  ) => (
+    <StringField
+      label={config === "columns" ? "" : field}
```

Comment:
Cool! I think the option of having no `label` should be an argument of the function, as opposed to hardcoding `config === "columns"` here. This is the simplest change I can think of, but it might also be possible to be cleaner:
```suggestion
    config?: SubConfigKeys,
    label = field
  ) => (
    <StringField
      label={label}
```
```diff
               {displayStringOnlyFields(
                 "text_input",
                 resultingConfig.columns.text_input,
-                "columns"
+                "columns",
+                ""
               )}
               <Typography variant="body2">label:</Typography>
               {displayStringOnlyFields(
                 "label",
                 resultingConfig.columns.label,
-                "columns"
+                "columns",
+                ""
               )}
```